net/mihomo/files/mihomo.conf

```diff
@@ -0,0 +1,6 @@
+config mihomo 'globals'
+	option enabled '0'
+	option syslog '1'
+	option user 'mihomo'
+	option workdir '/etc/mihomo/'
```

Comment:
I think resources that are barely modified can be put under /usr/share
`option workdir '/usr/share/mihomo'`